apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-modal.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import LocationAutoCompleteTextbox from './location-autocomplete-textbox';
+import { CustomTextInput } from '../../../components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons';
+import { MullBackButton } from '../../../components';
+import { EventRestriction, ILocation } from '@mull/types';
+import { FormikConfig } from 'formik';
+
+export interface LocationAutocompleteModalProps {
+  formik: FormikConfig<{
+    activeRestriction: EventRestriction;
+    startDate: any;
+    endDate: any;
+    startTime: string;
+    endTime: string;
+    eventTitle: string;
+    description: string;
+    location: ILocation;
+    imageFile: string;
+  }>;
+}
+
+export default function LocationAutocompleteModal({
+  formik: { touched, setFieldValue, errors, values },
+}) {
+  const [inputValue, setInputValue] = React.useState('');
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSetValue = (location: ILocation) => {
+    setInputValue(location.title);
+    setFieldValue('location', location);
+    handleClose();
+  };
+
+  return (
+    <React.Fragment>
```

Comment:
You're still using `<React.Fragment>` in your code (not just here). You can change it to `<>` to keep it consistent with the rest of our code

Reply:
Small error on our side this was done but not committed will be done in the next pushes